src/Fixer/Whitespace/StatementIndentationFixer.php

```diff
@@ -709,6 +701,37 @@ private function getLineIndentationWithBracesCompatibility(Tokens $tokens, int $
         return $regularIndent;
     }
 
+    /**
+     * Returns whether the token at given index is the last token in a property
+     * declaration before the type or the name of that property.
+     */
+    private function isPropertyStart(Tokens $tokens, int $index): bool
```

Comment:
This would be more helpful as a commonly-accessible helper method. I believe we have similar thing somewhere else already, and it's getting out of hand 😅. But it's good enough, I don't expect you finding better place for it at this point. I just think, in general, we need more reusable code for token traversing, manipulating etc.

Reply:
I agree but in this case the method is quite specific: we want the last token before the type or the property name. This is due to how the fixer is currently implemented and it wouldn't be surprising to see that changed in the future, so I'd keep it here for now.